eth2deposit/deposit.py

```diff
@@ -1,22 +1,45 @@
-import sys
 import click
+import sys
 
 from eth2deposit.cli.existing_mnemonic import existing_mnemonic
 from eth2deposit.cli.new_mnemonic import new_mnemonic
+from eth2deposit.utils.click import (
+    captive_prompt_callback,
+    choice_prompt_func,
+)
+from eth2deposit.utils import config
+from eth2deposit.utils.constants import INTL_LANG_OPTIONS
+from eth2deposit.utils.intl import (
+    get_first_options,
+    fuzzy_reverse_dict_lookup,
+    load_text,
+)
 
 
 def check_python_version() -> None:
     '''
     Checks that the python version running is sufficient and exits if not.
     '''
     if sys.version_info < (3, 7):
-        click.pause('Your python version is insufficient, please install version 3.7 or greater.')
+        click.pause(load_text(['err_python_version']))
         sys.exit()
 
 
 @click.group()
-def cli() -> None:
-    pass
+@click.pass_context
+@click.option(
+    '--language',
```

Comment:
Maybe update README.md to list and explain this argument?

Reply:
Resolved in https://github.com/ethereum/eth2.0-deposit-cli/pull/182/commits/e15806bac672f26e1b214626ed2371a69325167d